### ISSUE TITLE
Documentation pages not loading assets correctly

### ISSUE THREAD

**Issue by lizconlan** [mysociety/alaveteli#4109]
When I redeployed the docs site just now (after merging #4080 which only touches `/docs/glossary.md`), pages within folders (including the /es/ pages) were being built with the wrong assets path so were being rendered without CSS.

Instead of the expected path prefix from `{{site.baseurl}}` of '/', it was trying to load from the local folder path e.g. `/docs/assets/css/...` and 404ing. To avoid the docs pages appearing broken, I've made the simplest possible fix of changing the asset paths in the layout file to start with a literal '/'.

My working assumption is that something has changed in how paths are evaluated and that merging the changes to the glossary page triggered a complete rebuild of the gh-pages site rather than the changes themselves having caused the error directly. (Full disclosure - I did not visit the page immediately before deploying but as @davewhiteland included screenshots I am reasonably confident in assuming the pages were working as expected pre deploy.)

It would be good to allocate some time to investigate this fully and possibly come up with a better fix. 

***Note:*** images are currently broken on the Spanish pages as these also rely on the `{{site.baseurl}}/assets/` syntax

**Comment by lizconlan** [comment]
Possibly related:
* Jekyll ticket (closed): [GitHub Pages Project Pages - Relative Links Fail](https://github.com/jekyll/jekyll/issues/332)
* StackOverflow: [GitHub pages and relative paths](https://stackoverflow.com/questions/16316311/github-pages-and-relative-paths)
* GItHub blog post (Dec 2016, tagged "New Features"): [Relative links for GitHub pages](https://github.com/blog/2290-relative-links-for-github-pages)
* Jekyll docs: [Specific docs for GitHub Pages](https://jekyllrb.com/docs/github-pages/)

tl;dr - the GitHub blog post implies something changed since our last docs rebuild and the Jekyll docs seem to suggest `site.github.url` may be helpful, although I'm not sure what effect that would have on being able to build the docs pages locally

**Comment by stevenday** [comment]
https://github.com/github/pages-gem/issues/350#issuecomment-317429640 suggests that our current config setting of "/" for baseurl should be "" and then links should be refactored to include the "/" at the beginning of the path

**Comment by garethrees** [comment]
See https://github.com/jekyll/jekyll/issues/6057 via https://github.com/mysociety/alaveteli/issues/4130

**Comment by lizconlan** [comment]
Fixed in #4172
